### ISSUE TITLE
[4.0]Mystic Mana Wall

### ISSUE THREAD

**Issue by Alacanus** [ChalkyBrush/roshpit-bug-tracker#1382]
Additional Resistance not working

**Comment by Nintai** [comment]
Works just fine. Please test it before writing a bug report, then you would notice its "doesnt show in tooltip" and not "doesnt work"

**Comment by redledcat** [comment]
Yep, it's different from resitance, and either we close that issue, or teach tooltip to show it when mana is sufficient
